esp32c6-hal/Cargo.toml

```diff
@@ -33,30 +33,30 @@ embassy-time   = { version = "0.1.5",  features = ["nightly"], optional = true }
 aes                = "0.8.3"
 critical-section   = "1.1.2"
 crypto-bigint      = { version = "0.5.3", default-features = false}
-embassy-executor   = { version = "0.3.2", features = ["nightly", "integrated-timers", "arch-riscv32", "executor-thread"] }
+embassy-executor   = { version = "0.3.2", features = ["nightly"] }
 embedded-graphics  = "0.8.1"
 embedded-hal-1     = { version = "=1.0.0-rc.1", package = "embedded-hal" }
 embedded-hal-async = "=1.0.0-rc.1"
 embedded-can       = "0.4.1"
 embedded-io-async  = "0.6.0"
-esp-backtrace      = { version = "0.9.0", features = ["esp32c6", "panic-handler", "exception-handler", "print-uart"] }
+esp-backtrace      = { version = "0.9.0", features = ["esp32c6", "panic-handler", "exception-handler", "print-jtag-serial"] }
 esp-hal-smartled   = { version = "0.6.0", features = ["esp32c6"], path = "../esp-hal-smartled" }
-esp-println        = { version = "0.7.0", features = ["esp32c6"] }
+esp-println        = { version = "0.7.0", default-features=false, features = ["esp32c6", "jtag_serial"] }
```

Comment:
This is probably unintentional, but let's stick with UART for consistency please :)